modules/nf-core/glimpse2/chunk/meta.yml

```
@@ -0,0 +1,74 @@
+name: "glimpse2_chunk"
+description: Defines chunks where to run imputation
+keywords:
+  - chunk
+  - imputation
+tools:
+  - "glimpse2":
+      description: "GLIMPSE2 is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+requirement: AVX2
```

```suggestion
requirements: 
  - AVX2
```

Sorry, was just reading https://nfcore.slack.com/archives/CE6SDBX2A/p1677231429855929 to understand what's going on here. This is the first time this has been used, so let's be consistent with the other fields and make it a plural and a list.

Otherwise I think we're good, but I'll ask for a second review as there's some new stuff here.

---

No problem.
Thanks a lot for your review !